ci-operator/config/openshift/cluster-control-plane-machine-set-operator/openshift-cluster-control-plane-machine-set-operator-main.yaml

```diff
@@ -269,6 +273,18 @@ tests:
         requests:
           cpu: 100m
     workflow: ipi-nutanix
+- as: regression-clusterinfra-azure-ipi-cpms-allcases
+  optional: true
+  steps:
+    cluster_profile: azure4
```

Comment:
Are we picking Azure for a particular reason? Could this be AWS instead? Azure is one of the flakiest platforms, and AWS is more stable, but, if there's good reason to use Azure then happy to keep this way

Reply:
acked,changed to aws and adding for 4.15 and 4.16 as well.